### ISSUE TITLE
onnx::SoftmaxCrossEntropyLoss not supported

### ISSUE THREAD

**Issue by Sol-Bridger** [Verified-Intelligence/auto_LiRPA#41]
Hi,

I get the following error that onnx::SoftmaxCrossEntropyLoss is an unsupported operation for my Transformer network. 
I'm using the Huggingface GPT2ForSequenceClassification model (and the SST2 dataset) where there are two classes but no changes have been made to the underlying architecture.
Is the softmax cross entropy loss an unsupported operation altogether? If not, do you plan on supporting it in the future as it's commonly needed?
Also, is the argmax function also unsupported?

**System configuration:**
 - OS: Ubuntu 22.04.5
 - Python version: Python 3.7
 - Pytorch Version: PyTorch 1.12.1+cu102
 - Have you tried to reproduce the problem in a cleanly created conda/virtualenv environment using official installation instructions and the latest code on the main branch?: Yes
![Screenshot (12)](https://user-images.githubusercontent.com/116904575/217239055-80c81811-32cf-452c-b06b-4946b15db88b.png)

**Comment by shizhouxing** [comment]
Hi @Sol-Bridger , cross entropy loss is supported with our [CrossEntropyWrapper](https://github.com/Verified-Intelligence/auto_LiRPA/blob/master/auto_LiRPA/wrapper.py#L4). But currently our library is unlikely to work for `GPT2ForSequenceClassification` which may involve many operators defined in other ways. You may refer to our [example](https://github.com/Verified-Intelligence/auto_LiRPA/tree/master/examples/language) which has a simplified Transformer.
